www/app.py

```diff
@@ -3,10 +3,18 @@
 from os.path import isfile, join
 from flask_cors import CORS, cross_origin
 import xml.etree.ElementTree
+import Adafruit_GPIO.PWM as pwmLib
+import Adafruit_GPIO.GPIO as gpioLib
+import json
+import threading
 
 app = Flask(__name__)
 CORS(app)
 
+PWM_FREQ_HZ = 1000
```

Comment:
It could have been passed in when the PWM is started, but it looks like I had left it out and let it fall to the default speed. So, this line could be removed.